CMakeLists.txt

```diff
@@ -1,5 +1,23 @@
 cmake_minimum_required(VERSION 3.12)
-find_package(CUDA REQUIRED)
+# This has to be specified before enable_language according to
+# https://gitlab.kitware.com/cmake/cmake/issues/17559
```

Comment:
what is the context here? We use NVCC_FLAGS from CUDA package before, and we now remove that dependency?

Reply:
the context is the title, for reference, refer to
https://cliutils.gitlab.io/modern-cmake/chapters/packages/CUDA.html